config/jobs/kubeflow/kubeflow-presubmits.yaml

```diff
@@ -198,3 +198,109 @@ presubmits:
+        args:
+        - --clean
+        - --repo=github.com/kubeflow/$(REPO_NAME)=$(PULL_REFS)
+        - --scenario=execute
+        - -- # end bootstrap args, scenario args below
+        - -- # end kubernetes_execute_bazel flags (consider following flags as text)
+        - ./test/presubmit-tests.sh
+        - "--workflow_file"
+        - "build_image.yaml"
+    run_after_success:
+    - name: presubmit-e2e-test
+      always_run: true
+      skip_report: false
+      rerun_command: /test presubmit-e2e-test
+      trigger: (?m)^/test (all|presubmit-e2e-test),?(\s+|$)
+      labels:
+        preset-service-account: "true"
+      spec:
+        containers:
+        - image: gcr.io/k8s-testimages/kubekins-e2e:v20181024-be2f242dd-1.10
+          args:
+          - --clean
+          - --repo=github.com/kubeflow/$(REPO_NAME)=$(PULL_REFS)
+          - --scenario=execute
+          - -- # end bootstrap args, scenario args below
+          - -- # end kubernetes_execute_bazel flags (consider following flags as text)
+          - ./test/presubmit-tests.sh
+          - "--workflow_file"
+          - "e2e_test_gke.yaml"
+          - "--test_result_folder"
+          - "e2e_test"
+    - name: presubmit-e2e-test-gce-minikube
+      always_run: true
+      skip_report: true
+      rerun_command: /test presubmit-e2e-test-gce-minikube
+      trigger: (?m)^/test (all|presubmit-e2e-test-gce-minikube),?(\s+|$)
+      labels:
+        preset-service-account: "true"
+      spec:
+        containers:
+        - image: gcr.io/k8s-testimages/kubekins-e2e:v20181024-be2f242dd-1.10
+          args:
+          - --clean
+          - --repo=github.com/kubeflow/$(REPO_NAME)=$(PULL_REFS)
+          - --scenario=execute
+          - -- # end bootstrap args, scenario args below
+          - -- # end kubernetes_execute_bazel flags (consider following flags as text)
+          - ./test/presubmit-tests-gce-minikube.sh
+          - "--workflow_file"
+          - "e2e_test_gke.yaml"
+          - "--test_result_folder"
+          - "e2e_test"
+    - name: presubmit-sample-test
+      always_run: true
+      skip_report: true
+      rerun_command: /test presubmit-sample-test
+      trigger: (?m)^/test (all|presubmit-sample-test),?(\s+|$)
+      labels:
+        preset-service-account: "true"
+      spec:
+        containers:
+        - image: gcr.io/k8s-testimages/kubekins-e2e:v20181024-be2f242dd-1.10
+          args:
+          - --clean
+          - --repo=github.com/kubeflow/$(REPO_NAME)=$(PULL_REFS)
+          - --scenario=execute
+          - -- # end bootstrap args, scenario args below
+          - -- # end kubernetes_execute_bazel flags (consider following flags as text)
+          - ./test/presubmit-tests.sh
+          - "--workflow_file"
+          - "sample_test.yaml"
+          - "--test_result_folder"
+          - "sample_test"
+  - name: presubmit-unit-test
+    always_run: true
+    skip_report: false
+    rerun_command: /test presubmit-unit-test
```

Comment:
you can drop `rerun_command` and `trigger` section in OSS config now as they are default

Reply:
also maybe better to write the job with podutils now, and you can drop some more boilerplates, let me find you an example

Reply:
doen